build.ps1

```diff
@@ -90,17 +90,16 @@ $REPO = Get-Item "$PSScriptRoot" | select -ExpandProperty FullName
 $BUILD_DIR = "$REPO\_build"
 $SCRATCH_DIR = "$BUILD_DIR\scratch"
 $OUTPUT_DIR = "$BUILD_DIR\output"
-$OUTPUT_20_DIR = "$OUTPUT_DIR\netstandard2.0"
 $OUTPUT_48_DIR = "$OUTPUT_DIR\dotnet48"
+$OUTPUT_47_DIR = "$OUTPUT_DIR\dotnet47"
 $OUTPUT_46_DIR = "$OUTPUT_DIR\dotnet46"
-$OUTPUT_45_DIR = "$OUTPUT_DIR\dotnet45"
 $PATCHES = "$REPO\patches"
 
 Write-Host 'DEBUG: Printing MSBuild.exe version...'
 msbuild /ver
 Write-Host ''
 
-mkdirClean $BUILD_DIR, $SCRATCH_DIR, $OUTPUT_DIR, $OUTPUT_20_DIR, $OUTPUT_48_DIR, $OUTPUT_46_DIR, $OUTPUT_45_DIR
+mkdirClean $BUILD_DIR, $SCRATCH_DIR, $OUTPUT_DIR, $OUTPUT_47_DIR, $OUTPUT_46_DIR
```

Comment:
Whilst building the packages I received this error:
![image](https://github.com/user-attachments/assets/06e9e935-f35e-4493-802f-9d24e1f527c3)
And this is what the build output looked like.
![image](https://github.com/user-attachments/assets/2082592b-9943-4204-ba62-5189bac61742)

It seems that we need to add `$OUTPUT_48_DIR` to create a directory.

Reply:
Oops, I was meaning to add 47, not replace 48 🤦‍♀️